packages/react/src/components/Alert/Alert.tsx

```diff
@@ -153,17 +177,47 @@ class Alert extends AutoControlledComponent<WithAsProp<AlertProps>, AlertState>
   }
 
   renderContent = ({ styles, accessibility }: RenderResultConfig<AlertProps>) => {
-    const { actions, dismissible, dismissAction, content } = this.props
+    const { actions, dismissible, dismissAction, content, icon, header, body } = this.props
 
-    return (
+    const bodyContent = (
       <>
+        {Box.create(header, {
```

Comment:
Is it `Box` or `Text`?

https://github.com/stardust-ui/react/pull/1821/files#diff-55e5002251082bee3582b3d7cad2ceedR57

Reply:
Changed it to `Text`